### PR TITLE
[HERETICS] Void & a handful of heretics-related PRS

### PR DESCRIPTION
**Pull request by kiwedespars**
## About The Pull Request

ports void path for heretics, and a ton of other prs related to heretics from tg, along with a few additions/changes.

infinite void - after a 5 second windup, make a 7x7 area your temporary domain, slowing enemies inside and also marking them with a void mark.

((voids repulse (placeholder) - snapping fingers sends anything nearby flying back.)) goodbye.

void's applause - clap your hands and switch places with someone of your choice.

credits to 
tgstation/tgstation#54252
tgstation/tgstation#54371
tgstation/tgstation#54705
tgstation/tgstation#55050
tgstation/tgstation#55437
tgstation/tgstation#55630
tgstation/tgstation#55667
tgstation/tgstation#56268
tgstation/tgstation#56304
tgstation/tgstation#56441

## Why It's Good For The Game

more antag content good.

## Changelog
:cl:
add: New heretic path - Path of Void- it specializes in being extremely stealthy.
tweak : Removed curse of blindness replaced with mask of madness. hey it even rhymes.
fix: fixes heretic mass deletion during transmutation bug.
fix: Fixes heretic brews being permanent.
fix: Fixes void storm breaking after resurrecting.
tweak: Heretic has received a minor textual facelift.
tweak: Heretics who finish the Void Path and become an Aristocrat of the Void can now survive in the Void (space).
fix: Heretics who research Aristocrats Way on the Void Path will no longer suffocate in their own storm when they ascend (no longer breathes).
fix: Mark of Void and Seeking Blade are once again exclusive only to Void.
balance: Carving knife now deals more damage on throw and can embed in your enemies.
tweak: Grasp of Rust only rusts floors and machines on harm intent.
tweak: when u choose a sac target as heretic it ll also tell the job of the sac
add: 2 new void spells, one a placeholder.
add: flesh mansus grasp buffed, now gives you 5u eldritch fluid when hitting someone.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
Take note, you're accidentally deleting the succumb action button icon

**Comment by kiwedespars** [comment]
wait wh at

File: code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+/datum/eldritch_knowledge/final/void_final/on_life(mob/user)
+	. = ..()
+	if(!finished)
+		return
+
+	for(var/mob/living/carbon/livies in spiral_range(7,user)-user)
+		if(IS_HERETIC_MONSTER(livies) || IS_HERETIC(livies))
+			return
+		livies.silent += 1
```
Comment (necromanceranne) [inline code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm]: Put a clamp on this.

File: code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+/datum/eldritch_knowledge/void_grasp/on_eldritch_blade(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)  || H.has_status_effect(/datum/status_effect/eldritch/void)
+	if(!E)
+		return
+	E.on_effect()
+	H.silent += 3
```
Comment (necromanceranne) [inline code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm]: Here too if you would.

File: code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+/datum/eldritch_knowledge/void_grasp/on_mansus_grasp(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!iscarbon(target))
+		return
+	var/mob/living/carbon/carbon_target = target
+	var/turf/open/turfie = get_turf(carbon_target)
+	turfie.TakeTemperature(-20)
+	carbon_target.adjust_bodytemperature(-40)
+	carbon_target.silent += 4
```
Comment (necromanceranne) [inline code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm]: Clampies

**Pull request merged**